kratos/tests/cpp_tests/utilities/test_geometrical_projection_utilities.cpp

```diff
@@ -44,7 +44,7 @@ GeometryPointType::Pointer CreateLine3D2NForTestPoint2D()
 {
     GeometryPointType::PointsArrayType points;
     points.push_back(Kratos::make_shared<Point>(0.0, 0.0, 0.0));
-    points.push_back(Kratos::make_shared<Point>(1.0, 0.0, 0.0));
+    points.push_back(Kratos::make_shared<Point>(2.0, 0.0, 0.0));
```

Comment:
why do you change this?

Reply:
The old implementation wouldn't pass the proposed check